tests/c/trace_too_long_hwt.c

```diff
@@ -0,0 +1,46 @@
+// # This test checks that an overflowing PT buffer is caught at the point
```

Comment:
Is this test the same as the last one, just with a higher loop bound?

Reply:
In essence, yes: in order to overflow PT we need a *loooooong* trace.